tests/storage/migration.go

```diff
@@ -854,6 +857,8 @@ func createBlankDV(virtClient kubecli.KubevirtClient, ns, size string) *cdiv1.Da
 		libdv.WithBlankImageSource(),
 		libdv.WithStorage(libdv.StorageWithStorageClass(sc),
 			libdv.StorageWithVolumeSize(size),
+			libdv.StorageWithVolumeMode(k8sv1.PersistentVolumeFilesystem),
```

Comment:
I see createBlankDV/createDV is being reused in many tests, will all of them be fine with it exclusively creating FS PVCs?